src/types/participant.d.ts

```diff
@@ -0,0 +1,22 @@
+import {Mongoose} from 'mongoose';
+
+export interface participant {
+    _id: Mongoose.types.ObjectId;
+    name: string;
+    email: string;
+    phone: string;
+    firstPreference: string;
+    secondPreference: string;
+    status: string;
+    emailed: boolean;
+}
+
+export interface updateParticipant {
```

Comment:
Use `Partial` typescript utility type to make an optional copy of `participant` in `updateParticipant`